tests/devapps/DesktopTestApp/GlobalSuppressions.cs

```diff
@@ -0,0 +1,8 @@
+﻿// This file is used by Code Analysis to maintain SuppressMessage
```

Comment:
Instead of GlobalSuppressions and pragma disable in code, can these rules be disabled in tests editorconfig for all tests?